mypy_django_plugin/transformers/managers.py

```diff
@@ -71,15 +72,18 @@ def get_method_type_from_dynamic_manager(
     queryset_info = helpers.lookup_fully_qualified_typeinfo(api, queryset_fullname)
     assert queryset_info is not None
 
-    def get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[ProperType]:
+    def get_funcdef_type(definition: Node | None) -> Optional[ProperType]:
         # TODO: Handle @overload?
         if isinstance(definition, FuncBase) and not isinstance(definition, OverloadedFuncDef):
             return definition.type
         elif isinstance(definition, Decorator):
             return definition.func.type
         return None
 
-    method_type = get_funcdef_type(queryset_info.get_method(method_name))
+    base_that_has_method = _get_base_containing_method(queryset_info, method_name)
```

Comment:
```suggestion
    base_that_has_method = queryset_info.get_containing_type_info(method_name)
```

`_get_base_containing_method` is now a reimplementation of `TypeInfo.get_containing_type_info`

Reply:
oups you are right, fixed in my last commit